mmaction/datasets/pipelines/loading.py

```diff
@@ -717,6 +717,60 @@ def __repr__(self):
         return repr_str
 
 
+class FFmpegDecodeMotionVector(object):
+    """
+        Ref:
+            https://github.com/chaoyuaw/pytorch-coviar.
+    """
+
+    def __init__(self, residual=False, gop_size=16):
+        self.residual = residual
+        self.gop_size = gop_size
+
+    def _clip_and_scale(self, img, size):
+        return (img * (127.5 / size) + 128).astype(np.int32)
+
+    def __call__(self, results):
+        try:
+            import coviar.load
```

Comment:
The package is not available in pypi, not a good practice. 
Better extract useful codes from the project `coviar`.

Reply:
Useful code are wrote in C++, is it good to add c++ source code into our repo? Besides, I found a better way to extract moment vector, this `load` function will open the source video #frames times. But [this test code from auther](https://github.com/PyAV-Org/PyAV/pull/558/files#diff-14e642ea908c02e56935e4d3bd50e8751be6a6eab820660e291c437e3eab38aaR60) seems better, though coviar claimed this operation is not the bottleneck(gpu is the bottleneck).